eng/CHECKENFORCER

```diff
@@ -1,2 +1,50 @@
 format: v0.1-alpha
 minimumCheckRuns: 1
+timeout: 5
+message: >
+  This pull request is protected by [Check Enforcer](https://aka.ms/azsdk/check-enforcer).
+
+  # What is Check Enforcer?
+  
+  Check Enforcer helps ensure all pull requests are covered by at least one
+  check-run (typically an Azure Pipeline). When all check-runs associated
+  with this pull request pass then Check Enforcer itself will pass.
+  
+  # Why am I getting this message?
+  
+  You are getting this message because Check Enforcer did not detect any
+  check-runs being associated with this pull request within five minutes. This
+  may indicate that your pull request is not covered by any pipelines and so
+  Check Enforcer is correctly blocking the pull request being merged.
+  
+  # What should I do now?
+  
+  If the **check-enforcer** check-run is not passing and all other check-runs
+  associated with this PR are passing (excluding _license-cla_) then you could
+  try telling _Check Enforcer_ to evaluate your pull request again. You can
+  do this by adding a comment to this pull request as follows:
+  
+  ```/check-enforcer evaluate```
```

Comment:
Not that it matters much but I've defaulted to calling `reset` as it is fewer chars and I find it easier to type :)

Reply:
Hopefully you aren't doing it that often that the extra 3 characters causes RSI ;)